<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowConsultarVeiculo</class>
 <widget class="QMainWindow" name="MainWindowConsultarVeiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar Veiculo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditModelo">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>51</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>51</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>141</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor de compra:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlaca">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>81</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKilometragem">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>111</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>111</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>81</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarca">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>21</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>111</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Km</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorDeCompra">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>141</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSituacao">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>170</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Disponivel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Situação:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>259</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
